### DISCUSSION TITLE
Storage Terms of Service Template

### DISCUSSION THREAD

**Discussion by csarven** [solid/specification#577] [General]
**Status**: Draft. Logging and parking this consideration here for now. It can move elsewhere.

**Purpose**: This ToS template is intended for storage providers to consider adopting, in addition to their local laws, in the spirit of the Solid project.

**How to contribute**: make specific bullet point suggestions, one per comment, that's not already covered here.

Here is an example from https://solidcommunity.net/ that can be adapted for general use (in addition to insights from other ToS from other servers and services):

> This system is intended for developers and early adventurous users to try using the Solid platform as we continue to develop and refine it. Solidcommunity.net is run by a loose collection of volunteers. We reserve the right to change or modify this policy at any time. 
> 
> To use this system, you must understand that we cannot make guarantees of any sort with respect to the security and privacy of data that you may store in a solidcommunity.net Solid pod or with respect to the functionality and availability of the system. 
> 
> Consequently, Solid Pods on Solidcommunity.net should not be used to store personal data, or any data that is otherwise sensitive, private, confidential, or intended in any way for commercial use. 
> 
> When you use Solid, information is stored in your Solid pod and in the Solid pods of other users, depending on the actions you take and the apps you use. You and other Solid pod users control who has access to that information. Some information, like your solid profile, is public. 
> 
> As a user of solidcommunity.net, you have the responsibility to understand what data you are storing in your Pod, whom that data is shared with, and what, if any, data you have made public. 
> 
> If you add public information to Solid then you must expect this information to be visible to anyone. You must ensure that such data is appropriate, as set out below, and also suitable for the legal jurisdictions and cultural communities in which you are connected. Information shared with a limited set of people must also be written responsibly and read discerningly. 
> 
> By using solidcommunity.net, you agree not to misuse the system or to assist or enable anyone else to misuse the system. For example, you must not do, or attempt to do, any of the following in connection with solidcommunity.net services: 
> 
> * violate the law in any way, including by storing, publishing or sharing illegal material, including material that's fraudulent, defamatory, or misleading;
> 
> * breach or otherwise circumvent any security or authentication measures;
> 
> * interfere with or disrupt any user, host, or network, for example by sending a virus, overloading, flooding, spamming, or mail-bombing any part of solidcommunity.net;
> 
> * access, search, or create accounts for solidcommunity.net by any means other than our publicly supported interfaces (for example, by "scraping" or creating accounts in bulk);
> 
> * send unsolicited communications, promotions or advertisements, or spam;
> 
> * send altered, deceptive or false source-identifying information, including "spoofing" or "phishing";
> 
> * promote or advertise products or services other than your own without appropriate authorization;
> 
> * abuse referrals or promotions to get more storage space than deserved;
> 
> * circumvent storage space limits;
> 
> * sell the services of solidcommunity.net unless specifically authorized to do so;
> 
> * harass or abuse solidcommunity.net system operators, representatives or agents performing services on behalf of the Solid Project;
> 
> * violate the privacy or infringe the rights of others;
> 
> * Use solidcommunity.net to violate any law or regulation. 
> 
> In the event that any user abuses or misuses the system, assists anyone else to abuse or misuse the system, or monopolizes an unfair proportion of available system or network resources, we reserve the right to limit that user’s activity or close and delete that user’s account. Such acts may also give rise to civil or criminal consequences.

**Comment by Charlotte-br560** [comment]
Hi there! Your draft for the ToS template is already quite comprehensive. Here are some specific suggestions:

Consider adding a section about user responsibilities, emphasizing the importance of securing their login credentials and taking necessary precautions to protect their data.

Include a disclaimer about the use of third-party applications or services with Solid pods and any potential risks associated with integrating external tools.

Address the procedures and policies related to handling user complaints, feedback, and reported issues, providing transparency on how these matters are addressed by the service provider.

Add a section about the process and conditions under which the Terms of Service may be updated or modified. Clearly communicate how users will be informed of such changes.

Mention the mechanism for handling legal requests, such as subpoenas or court orders, and the extent to which user data may be disclosed in compliance with the law.

Provide information on data retention policies, specifying how long user data will be stored and the circumstances under which it may be deleted.

Feel free to integrate these suggestions as separate bullet points into your ToS template draft. Adjustments based on the specific nature of your service and legal requirements are recommended. Good luck!
